components/create_post/create_post.jsx

```diff
@@ -156,7 +156,7 @@ export default class CreatePost extends React.Component {
     }
 
     static defaultProps = {
-        currentChannelMembersCount: 0,
+        currentChannelMembersCount: 1,
```

Comment:
@saturninoabril I changed this to 1 as the user is a member at the least. let me know if you want me to revert this back to 0

Reply:
Yeah, I think so. Same as in mattermost-redux.